tests/test_utils.py

```diff
@@ -39,8 +41,59 @@ def synapse_store():
         synapse_store = SynapseStorage()
     yield synapse_store
 
-
 class TestGeneral:
+    def test_clear_synapse_cache(self, tmp_path):
```

Comment:
Create a mock file under mock folder .synapseCache and make sure that cache can be cleared correctly using `clear_synapse_cache` function.